### DISCUSSION TITLE
vcruntime140.dll laragon full error in window 11

### DISCUSSION THREAD

**Discussion by DavidLuceroSigcho** [leokhoa/laragon#607] [Q&A]
<img width="507" alt="Captura de pantalla 2023-11-15 102317" src="https://github.com/leokhoa/laragon/assets/14005639/684f078c-6650-458a-87c4-df8019a3e5c7">
I have downloaded laragon full but at the moment of starting everything, I get that error that I attach the image.

**Comment by Pen-y-Fan** [comment]
The full version of Laragon normally installs the Visual Studio runtime. The first thing I would suggest is to restart Windows. If Laragon still fails download **vc_redist_x64** from [Apache lounge](https://www.apachelounge.com/download/).
